src/App.js

```diff
@@ -1,86 +1,214 @@
+        {(todos.length > 0 && todos.every(todo => todo.isCompleted === true))
+          ? (
+            <input
+              checked
+              type="checkbox"
+              id="toggle-all"
+              className="toggle-all"
+              onChange={e => toggleAll(e)}
+              value={allStatus}
+            />
+          )
+          : (
+            <input
+              type="checkbox"
+              id="toggle-all"
+              className="toggle-all"
+              onChange={e => toggleAll(e)}
+              value={allStatus}
+            />
+          )
+        }
         <label htmlFor="toggle-all">Mark all as complete</label>
 
-        <ul className="todo-list">
-          <li>
-            <div className="view">
-              <input type="checkbox" className="toggle" />
-              <label>asdfghj</label>
-              <button type="button" className="destroy" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li className="completed">
-            <div className="view">
-              <input type="checkbox" className="toggle" />
-              <label>qwertyuio</label>
-              <button type="button" className="destroy" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li className="editing">
-            <div className="view">
-              <input type="checkbox" className="toggle" />
-              <label>zxcvbnm</label>
-              <button type="button" className="destroy" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li>
-            <div className="view">
-              <input type="checkbox" className="toggle" />
-              <label>1234567890</label>
-              <button type="button" className="destroy" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-        </ul>
+        <TodoList
+          filter={filter}
+          handleEnter={handleEnter}
+          handleEditingTodo={handleEditingTodo}
+          removeItem={removeItem}
+          toggleCompletedStatus={toggleCompletedStatus}
+          todos={todosOnPage}
+          handleEscape={handleEscape}
+        />
       </section>
 
-      <footer className="footer">
-        <span className="todo-count">
-          3 items left
-        </span>
-
-        <ul className="filters">
-          <li>
-            <a href="#/" className="selected">All</a>
-          </li>
-
-          <li>
-            <a href="#/active">Active</a>
-          </li>
-
-          <li>
-            <a href="#/completed">Completed</a>
-          </li>
-        </ul>
-
-        <button type="button" className="clear-completed">
-          Clear completed
-        </button>
-      </footer>
+      {(todos.length >= 1) && (
+        <Footer
+          filter={filter}
+          clearCompleted={clearCompleted}
+          todos={todos}
```

Comment:
I'm not sure how to implement this solution since I don't store ActiveTodos anywhere but I need to show the length of exactly Active todos. So I pass all of them, then if Footer I'm filtering active todos array from all of them and print its length.